awx/main/models/events.py

```diff
@@ -546,6 +546,17 @@ def _update_host_summary_from_stats(self, hostnames):
 
             Host.objects.bulk_update(list(updated_hosts), ['last_job_id', 'last_job_host_summary_id'], batch_size=100)
 
+            updated_hosts_list = list()
+            for host in updated_hosts:
+                updated_hosts_list.append(host.name)
```

Comment:
Do we need this data structure, or can we just do `hostname__in=updated_hosts`.

If it *must* be a `list`, perhaps we could just do:

`hostname__in=list(updated_hosts)` instead of doing the loop here?

Reply:
```hostname__in``` has to be list of strings. It has to be list of string because the schema has a column ```hostname``` which is a string. From what my understanding is , i can only filter it against list of strings. 

```hostname__in=list(updated_hosts)``` would make ```hostname__in``` a list of ```Host``` ,and cant be used to filter the table.